biocrnpyler/species.py

```diff
@@ -784,8 +777,7 @@ def name(self):
 
     def __hash__(self):
         ophash = OrderedPolymer.__hash__(self)
-        ophash += hash(self.circular)+hash(self.base_species) + \
-            hash(self.name)+hash(self.material_type)
+        ophash += hash(self.circular)+hash(self.name)+hash(self.material_type)
```

Comment:
This is missing attributes. Maybe we should make the ophash just be `hash(str(self))`